tests/virt/cluster/service_account/test_service_account.py

```diff
@@ -46,10 +48,13 @@ def test_vm_with_specified_service_account(service_account_vm):
     assert pod_sa == vm_namespace, "ServiceAccount should be attached to the POD"
 
     # Verifies that ServiceAccount is attached to VMI
+    # Change mount device based on cluster cpu architecture
+    node_cpu_arch = infra.get_nodes_cpu_architecture()
+    mount_device = "/dev/vdb" if node_cpu_arch == S390X else "/dev/sda"
```

Comment:
`node_cpu_arch` used once, then use it directly.
```suggestion
    mount_device = "/dev/vdb" if infra.get_nodes_cpu_architecture() == S390X else "/dev/sda"
```